### DISCUSSION TITLE
deye_2mppt battery configuration

### DISCUSSION THREAD

**Discussion by 011V32** [davidrapan/ha-solarman#147] [Q&A] [answered]
Hello.
I switched here from [StephanJoubert](https://github.com/StephanJoubert/home_assistant_solarman) and am currently configuring it. I have a Deye SUN800G3-EU-230 and would have chosen the deye_2mppt.yaml as before. However, now you have to configure a battery, which my inverter doesn't support at all.
How do I go about this?

I also wanted to ask if there is a description of the configuration somewhere? (Similar to the original repo.)
I also ask myself what the "Solarman protocol passthrough mode" is for?
Thanks.

**Comment by davidrapan** [comment, marked as answer]
You just leave it as it is. Those configurables are only for profiles with batteries. 😉

**Reply by 011V32** [reply]
OK. thx.
And what is the "Solarman protocol passthrough mode" for?

**Reply by davidrapan** [reply]
For Ethernet version of the stick logger.

**Reply by 011V32** [reply]
What does this mean in my case?
(I have an inverter that has WiFi integrated.)

**Reply by davidrapan** [reply]
Nothing. Just ignore that.

**Reply by 011V32** [reply]
Some Information/documentation on that would be fine 😉
